<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Libi" ignored="0" name="batch1" tests="6" failures="1" timestamp="15 May 2018 20:51:18 GMT" time="123.767" errors="0">
  <testcase name="@BeforeMethod verifyHomepageTitle" time="1.259" classname="TestNG.Assertinon">
    <failure type="org.openqa.selenium.SessionNotCreatedException" message="Tried to run command without establishing a connection
Build info: version: &amp;apos;3.8.1&amp;apos;, revision: &amp;apos;6e95a6684b&amp;apos;, time: &amp;apos;2017-12-01T19:05:14.666Z&amp;apos;
System info: host: &amp;apos;LIBI&amp;apos;, ip: &amp;apos;10.1.10.42&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_77&amp;apos;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 59.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 2008, moz:profile: C:\Users\molla\AppData\Loca..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 313f3287-f9ef-4ee2-b9b2-394759d7bc6c">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Tried to run command without establishing a connection
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'
System info: host: 'LIBI', ip: '10.1.10.42', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 59.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 2008, moz:profile: C:\Users\molla\AppData\Loca..., moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 10.0, rotatable: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: 313f3287-f9ef-4ee2-b9b2-394759d7bc6c
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:658)
at org.openqa.selenium.remote.RemoteWebDriver.getTitle(RemoteWebDriver.java:329)
at TestNG.Assertinon.verifyHomepageTitle(Assertinon.java:44)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod verifyHomepageTitle -->
  <testcase name="@AfterMethod goBackToHomepage" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- @AfterMethod goBackToHomepage -->
  <testcase name="@BeforeMethod verifyHomepageTitle" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- @BeforeMethod verifyHomepageTitle -->
  <testcase name="@AfterMethod goBackToHomepage" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- @AfterMethod goBackToHomepage -->
  <testcase name="@AfterTest aftertest" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- @AfterTest aftertest -->
  <testcase name="@AfterTest aftertest" time="0.0" classname="TestNG.BeforeAfter2">
    <skipped/>
  </testcase> <!-- @AfterTest aftertest -->
  <testcase name="forgot" time="5.173" classname="TestNG.BeforeAfter2"/>
  <testcase name="link" time="12.302" classname="TestNG.BeforeAfter2"/>
  <testcase name="login" time="11.042" classname="TestNG.BeforeAfter2">
    <failure type="java.lang.AssertionError" message="expected [Facebook - Log In or Sign Up] but found [Log into Facebook | Facebook]">
      <![CDATA[java.lang.AssertionError: expected [Facebook - Log In or Sign Up] but found [Log into Facebook | Facebook]
at TestNG.BeforeAfter2.login(BeforeAfter2.java:42)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
at java.lang.Thread.run(Thread.java:745)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="assert_function" time="0.001" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- assert_function -->
  <testcase name="forgot" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- forgot -->
  <testcase name="login" time="0.0" classname="TestNG.Assertinon">
    <skipped/>
  </testcase> <!-- login -->
</testsuite> <!-- batch1 -->
